lib/ClimaCoreVTK/src/ClimaCoreVTK.jl

```diff
@@ -100,11 +100,11 @@ end
         basename::String,
         times,
         fields,
-        [ispace=vtk_space(fields)];
+        [ispace=vtk_space(first(fields))];
         vtkargs...
     )
 
-Write a sequence of fields `fields` at times `times` as a Paraview collection.
```

Comment:
This format is specific to paraview

Reply:
In that case, we should probably rename the function (`writepvd`?)

Reply:
we can do that in a separate PR.

Reply:
I think that might be less confusing, it looks like they are also pushing the new JSON series format, so that would be `write_vtk_series` or something similar.

Reply:
I can put the docstring back as it was. Yeah, I was misled by the name of the function and automatically assumed that it wasn't Paraview-specific